spec/requests/user_subscriptions_spec.rb

```diff
@@ -0,0 +1,131 @@
+require "rails_helper"
+
+RSpec.describe "UserSubscriptions", type: :request do
+  # TODO: (Alex Smith) remove super_admin restriction before final release
```

Comment:
See my [other comment](https://github.com/thepracticaldev/dev.to/pull/8779/files#r442573640) for more details. I realize in the model it technically allows `admin` _OR_ `super_admin`...since this is going to be removed anyways in the next 2 or 3 PRs and before we officially release it, I didn't go through the trouble of testing both roles. I simply added it to the spec so all my specs are still relevant and work - I just have to remove the `super_admin` in the PR where I setup roles and permissions.